static/app/components/events/interfaces/spans/gapSpanDetails.tsx

```diff
@@ -8,23 +8,23 @@ import {SectionHeading} from 'sentry/components/charts/styles';
 import ExternalLink from 'sentry/components/links/externalLink';
 import {FlamegraphPreview} from 'sentry/components/profiling/flamegraph/flamegraphPreview';
 import QuestionTooltip from 'sentry/components/questionTooltip';
+import {useFlamegraphTheme} from 'sentry/domains/profiling/hooks/useFlamegraphTheme';
+import {FlamegraphThemeProvider} from 'sentry/domains/profiling/providers/flamegraphThemeProvider';
+import {useProfileGroup} from 'sentry/domains/profiling/providers/profileGroupProvider';
```

Comment:
Any reason why the flamegraph theme provider and hook are split into the providers and hooks folders but the profile group hook is instill in the provider folder?

Reply:
I havent paid much attention to all the dir, it was mostly moving stuff to one of the top level folders, def should/will be addressed